From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Tue, 15 Mar 2016 10:42:16 -0400
Subject: fix -Wformat

---
 sscanApp/src/saveData.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sscanApp/src/saveData.c b/sscanApp/src/saveData.c
index 2984424..ccac1c2 100644
--- a/sscanApp/src/saveData.c
+++ b/sscanApp/src/saveData.c
@@ -843,7 +843,7 @@ void saveData_Info() {
 		printf("  links:");
 		cur= scan;
 		while (cur) {
-			printf(cur->name);
+			printf("%s", cur->name);
 			cur= cur->nxt;
 			if (cur) printf("->");
 		}
