From: Martin Konrad <konrad@frib.msu.edu>
Date: Fri, 11 Jan 2019 13:33:26 -0500
Subject: Add missing format string

---
 sscanApp/src/saveData_writeXDR.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sscanApp/src/saveData_writeXDR.c b/sscanApp/src/saveData_writeXDR.c
index d2b4715..2d0451b 100644
--- a/sscanApp/src/saveData_writeXDR.c
+++ b/sscanApp/src/saveData_writeXDR.c
@@ -851,7 +851,7 @@ void saveData_Info() {
 		printf("  links:");
 		cur= scan;
 		while (cur) {
-			printf(cur->name);
+			printf("%s", cur->name);
 			cur= cur->nxt;
 			if (cur) printf("->");
 		}
